router/tests/integration/CMakeLists.txt

```diff
@@ -20,6 +20,10 @@
 # along with this program; if not, write to the Free Software
 # Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 
+IF(DEBUG_EXTNAME)
+  ADD_COMPILE_DEFINITIONS(WITH_DEBUG_EXTNAME)
```

Comment:
Please rename `WITH_DEBUG_EXTNAME` to `HAVE_DEBUG_EXTNAME`.
`WITH_XXX` are usually CMake option not preprocessor defines.

Reply:
Done